internal/chain/engine.go

```diff
@@ -312,6 +310,14 @@ func payoutUndelegations(
 	return nil
 }
 
+// IsCommitteeSelectionBlock checks if the given header is for the committee selection block
+// which can only occur on beacon chain and in pre-staking era
```

Comment:
saying "after staking is enabled (epoch > pre-staking epoch)" is less confusing then "in pre-staking era."

Reply:
agreed, I changed the comment